import_cambridge_lands.py

```diff
@@ -144,7 +144,7 @@ def import_cambridge(
     3. Run this import
     '''
     command = CambridgeLandsImportCommand(
-        filename, apiurl, apitoken, lrapiurl, lrtoken, voaapiurl, voatoken)
+        filenames, apiurl, apitoken, lrapiurl, lrtoken, voaapiurl, voatoken)
```

Comment:
are you going to change the function this calls from `file_name` to `file_names` too?